app/public/human/human.js

```diff
@@ -1,34 +1,54 @@
 const HUMAN = (() => {
+  const enterRoom = (room) => {
+    WS.send({
+      action: {
+        name: "enterRoom",
+        args: {
+          room: room,
+        },
+      },
+      username: USERNAME_COOKIE,
+      gameId: GAME_ID_COOKIE,
+    });
+  };
 
-    const enterRoom = (room) => {
-        WS.send({
-            action: {
-                name: "enterRoom",
-                args: {
-                    room: room
-                }
-            },
-            username: USERNAME_COOKIE,
-            gameId: GAME_ID_COOKIE
-        });
-    };
+  const useTool = (tool) => {
+    WS.send({
+      action: {
+        name: "useTool",
```

Comment:
I recommend changing this to "switchTool". Remember, using a tool is automatic when entering a threatened room with the right tool